config/workload.yaml

```diff
@@ -1,11 +1,12 @@
-apiVersion: kontinue.io/v1alpha1
+apiVersion: carto.run/v1alpha1
 kind: Workload
 metadata:
   name: sample-app-java
   labels:
     app.tanzu.vmware.com/workload-type: web
-  run-image: harbor-repo.vmware.com/tanzu_desktop/sample-app-java
```

Comment:
Oh do we no longer need a separate run-image reference for Tilt to update? I thought that was so that Tilt is going to mutate that field to sample-app-java:sha-some-tilt-tag, and we don't want it to be mutating a field like the spec.image that's actually being used by the supplychain.

Reply:
Ming moved it to spec.params

Reply:
I moved it to spec.params
having run-image in metadata causes a validation failure on k8s client v1.21.2

Reply:
oops missed that part.